statsmodels/compat/pandas.py

```diff
@@ -2,47 +2,21 @@
 
 from distutils.version import LooseVersion
 
-import numpy as np
 import pandas
+from pandas import RangeIndex, Float64Index  # noqa:F401
 
 
 version = LooseVersion(pandas.__version__)
 
-if version >= LooseVersion('0.17.0'):
-    def sort_values(df, *args, **kwargs):
-        return df.sort_values(*args, **kwargs)
-else:
-    def sort_values(df, *args, **kwargs):
-        # always set inplace with 'False' as default
-        kwargs.setdefault('inplace', False)
-        return df.sort(*args, **kwargs)
 
-try:
-    from pandas import RangeIndex
-except ImportError:
-    RangeIndex = tuple()
+def sort_values(df, *args, **kwargs):
```

Comment:
Same here -- we shouldn't be calling  compat.sort_values (df) but df.sort_values()

Reply:
I agree, prefer to do this in a subsequent pass as the diff would touch a bunch of other files.